cmd/keytransparency-client/cmd/hammer.go

```diff
@@ -0,0 +1,219 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/google/keytransparency/core/authentication"
+
+	"github.com/aybabtme/uniplot/histogram"
+	"github.com/paulbellamy/ratecounter"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
```

Comment:
Are these new dependencies or is KT already using them?

Reply:
Key Transparency is already using the `spf13` dependencies.  The `ratecounter` and `histogram` dependencies are new. I should also point out that we are not importing the `cmd` directory - only the `core` directory.

Reply:
Ack.